db/migrate/20210208105018_clean_up_international_addresses_data.rb

```diff
@@ -0,0 +1,26 @@
+class CleanUpInternationalAddressesData < ActiveRecord::Migration[6.0]
+  def change
+    application_forms = ApplicationForm
+                        .international
+                        .where
+                        .not(postcode: nil)
+
+    audit_comment = 'Backfilled while removing the international addresses feature flag. International addresses should not be able to provide a postcode.'
+
+    application_forms.each do |application_form|
```

Comment:
```suggestion
    application_forms.find_each do |application_form|
```
though we may not yet have enough international addresses to be worried about this.

Reply:
still good to get in the habit. Thanks 👍